src/Dialect/ONNX/Transforms/ConstProp.cpp

```diff
@@ -134,6 +145,46 @@ Value ConstZeroTensor(
           type, rewriter.getZeroAttr(type.getElementType())));
 }
 
+template <typename GetFPConstFunc =
+              std::function<APFloat(const llvm::fltSemantics &, bool)>,
+    typename GetIntConstFunc = std::function<APInt(unsigned)>>
+Value GetClipConstantOfType(PatternRewriter &rewriter, ShapedType type,
```

Comment:
Could you lowercase the first character of a function? e.g. `getClipConstantOfType`. Same for the other functions:  CreateMaximumValueForClip, CreateMinimumValueForClip. Thanks!